runtime/src/test_utils.rs

```diff
@@ -317,10 +317,6 @@ impl Expectations {
             this.expect_emitted_events
         );
     }
-
-    fn skip_verification_on_drop(&mut self) {
```

Comment:
I never understood what this "skip verification" stuff was about, so I never landed it in `master`. I'm proposing dropping it from `next` in this PR, but maybe we want it for some reason?

Reply:
I believe this was just a merge thing. We already apply this when we call `reset`.